5_ModelsContinued.md

```diff
@@ -33,7 +33,7 @@ Add the following to order_form.stache:
 
     {{/each}}
 
-    {{#delivery}}
+    {{#each delivery}}
```

Comment:
`{{#each}}` is better than `{{#block}}` because it makes the fewest number of DOM changes possible.